### PR TITLE
Calendar Feature implemented

### PR DESCRIPTION
**Pull request by LittleJennie**

### CODE REVIEW

File: client/components/CalendarDays.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import moment from 'moment';
+
+const CalendarDays = () => {
+  const weekdaysshort = moment.weekdaysShort();
```
Comment (primanbo) [inline client/components/CalendarDays.jsx]: Camel-case variable names
```suggestion
  const weekdaysShort = moment.weekdaysShort();
```

Comment (primanbo) [reply client/components/CalendarDays.jsx]: also line 7

**Review by primanbo — COMMENTED** [review]
Great work, Jennie!

File: client/components/CalendarDays.jsx (hunk 1 above)
Comment (LittleJennie) [reply client/components/CalendarDays.jsx]: variable naming correct.

**Pull request merged**